deploy/examples/import-external-cluster.sh

```diff
@@ -78,11 +78,50 @@ function createClusterNamespace() {
   fi
 }
 
+function createRadosNamespaceCR() {
+  if ! kubectl -n "$NAMESPACE" get CephBlockPoolRadosNamespace $RADOS_NAMESPACE &>/dev/null; then
+    cat <<eof | kubectl create -f -
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPoolRadosNamespace
+metadata:
+  name: $RADOS_NAMESPACE
+  namespace: $NAMESPACE # namespace:cluster
+spec:
+  # blockPoolName is the name of the CephBlockPool CR where the namespace will be created.
+  blockPoolName: $RBD_POOL_NAME
+eof
+  else
+    echo "radosnamespace $RADOS_NAMESPACE already exists"
+  fi
+}
+
+function createSubvolumeGroupCR() {
+  if ! kubectl -n "$NAMESPACE" get CephFilesystemSubVolumeGroup $SUBVOLUME_GROUP &>/dev/null; then
+    cat <<eof | kubectl create -f -
+---
+apiVersion: ceph.rook.io/v1
+kind: CephFilesystemSubVolumeGroup
+metadata:
+  name: $SUBVOLUME_GROUP
+  namespace: $NAMESPACE # namespace:cluster
+spec:
+  # filesystemName is the metadata name of the CephFilesystem CR where the subvolume group will be created
+  filesystemName: $CEPHFS_FS_NAME
+eof
+  else
+    echo "subvolumegroup $SUBVOLUME_GROUP already exists"
+  fi
+}
+
 function importClusterID() {
   if [ -n "$RADOS_NAMESPACE" ]; then
+    createRadosNamespaceCR
+    sleep 10
```

Comment:
what are we waiting for here? a comment could be helpful :) .. same for the below line

Reply:
Right, hopefully we can wait for the namespace to actually be created, instead of a sleep

Reply:
Updated